### PR TITLE
update README.md and `treehouses help`

### LINKED ISSUE #863: README.md & help.sh add services commands
Perhaps we should do this? 
I think we should maybe use <> or some other character to differentiate between commands and service names. Cheers.

### PR DESCRIPTION
**Pull request by LordJashin32**
Fixes #863

### CODE REVIEW

**Comment by LordJashin32** [comment]
> Looks good. But makes me think that we need to rewrite our README.md soon. The current format is not very good.

yeah agreed. Going to make an issue for that.

File: README.md [hunk 1]
```diff
@@ -92,6 +92,9 @@ services [service_name] [command]         executes the given command on the spec
            [ntopng]                       Ntopng is a network traffic probe that monitors network usage
            [couchdb]                      CouchDB is an open-source document-oriented NoSQL database, implemented in Erlang
            [mariadb]                      MariaDB is a community-developed fork of the MySQL relational database management system
```
Comment (dogi) [inline README.md]: needs also to be changed

Comment (dogi) [reply README.md]: not just in the `help.sh`

**Comment by LordJashin32** [comment]
This new format looks even better I think. Less lines too. Cheers.

**Pull request merged**